examples/flux-todomvc/js/actions/TodoActions.js

```diff
@@ -42,17 +42,14 @@ var TodoActions = {
    */
   toggleComplete: function(todo) {
     var id = todo.id;
-    if (todo.complete) {
-      AppDispatcher.dispatch({
-        actionType: TodoConstants.TODO_UNDO_COMPLETE,
-        id: id
-      });
-    } else {
-      AppDispatcher.dispatch({
-        actionType: TodoConstants.TODO_COMPLETE,
-        id: id
-      });
-    }
+    var todoComplete = todo.complete ?
```

Comment:
Naming here is confusing.  We can just name this variable `actionType`, or we can move both id and actionType down to be inline in the definition of the action object itself.